File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-0.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int k = 1; 
        int prev = nums[0]; 
        for (int i = 1; i < nums.length; i++) {
            if (nums[i] != prev) {
                nums[k] = nums[i];
                prev = nums[i];
                k++;
            }
        }
        return k;
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        if len(nums) == 0:
            return 0
        longest = 1
        set_nums = set(nums)
        for num in nums:
            if num - 1 in set_nums:
                continue
            else:
                sum = 1
                next_num = num + 1
                while next_num in set_nums:
                    sum = sum + 1
                    if longest < sum:
                        longest = sum
                    next_num = next_num + 1
        return longest



        

